src/index.html

```diff
@@ -9,10 +12,64 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/page.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/stars__star.css"
     />
```

Comment:
Each attribute of the `link` tags should be on a new line with 2-space indentation. The closing bracket should be aligned with the opening tag.